### DISCUSSION TITLE
Sunflower Land Referral Competition

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2478] [Ideas]
How are you sure that the expected objectives will be achieved?
How many players are expected in 1 month?
How are you sure this won't be manipulated?
The objective for 1st must be at least 1.200 new players with 2 syncs.

**Comment by CarrotCounter** [comment]
Thanks for the thread @future2back 

We are doing the following to meet our objectives:
- Giving live updates on the weekly "Let's Grow Sunflower Land" Show on Discord. We will be giving additional prizes to participants during the show to increase engagement. 
- Listening to feedback about the competition from the community during this weekly show.
- Using the following leaderboard to track the winners: https://dune.com/sunflowerland/sunflower-land-referral-program 
- Incentivising overall winners with amazing prizes. 
- Utilising Twitter to further amplify the competition.

Our estimated new users is estimated 5k to 10k.

Estimate calculated as:
- 15k DAU 
- 20% uptake from community
- 2 to 3 referrals per participant
_(Estimate only and results may obviously differ)_

We maintain our policy of no multi-accounting. We utilise our usual multi-accounting strategies, and team can see easily see those who are manipulating the results. Those that do, will be disqualified. 

Thanks for opening the thread, and would love to hear your thoughts on how to increase engagement on the competition.

**Reply by future2back** [reply]
Thanks for replying bro @CarrotCounter.

First of all, how do you ensure people won't buy entry to new users who won't keep playing?


What advantage does a player recommended by someone have? Why don't we offer at least a welcome package to these players? 
Like:
- 1 T-Shirt _"Let's Grow Sunflower Land Event"_
- 5 SFL
- 20 Sunflower / Potato / Pumpkin
- 5 Wood
- 1 Sunflower Cake

**Comment by CarrotCounter** [comment]
Unfortunately, we can't force people to play Sunflower Land. For referrals we rely on the referrer to introduce their friend/ family to the game because they enjoy it. 

We accept that some players won't continue playing but we know some will most definitely continue playing because of the engaging game-play/ content and amazing community. 

Thank you for the suggestion. We need to be careful about giving more rewards because unfortunately people will abuse it. After the referral competition we will assess how we can improve. 

Thanks!  :)

**Reply by future2back** [reply]
Is the current result not concerning or is it within the expected range for the team?

Thank you.
